File: Demo Storyboard/LGSideMenuControllerDemo/LeftViewCell.m
//
//  LeftViewCell.m
//  LGSideMenuControllerDemo
//
//  Created by Friend_LGA on 26.04.15.
//  Copyright (c) 2015 Grigory Lutkov. All rights reserved.
//

#import "LeftViewCell.h"

@implementation LeftViewCell

- (void)awakeFromNib {
    [super awakeFromNib];

    // -----

    self.backgroundColor = [UIColor clearColor];

    self.textLabel.font = [UIFont boldSystemFontOfSize:16.f];
}

- (void)layoutSubviews {
    [super layoutSubviews];

    self.textLabel.textColor = self.tintColor;
    _separatorView.backgroundColor = [self.tintColor colorWithAlphaComponent:0.4];
}

- (void)setHighlighted:(BOOL)highlighted animated:(BOOL)animated {
    self.textLabel.textColor = highlighted ? [UIColor colorWithRed:0.f green:0.5 blue:1.f alpha:1.f] : self.tintColor;
}

@end

File: Demo Swift Storyboard/LGSideMenuControllerDemo/LeftViewCell.swift
//
//  LeftViewCell.swift
//  LGSideMenuControllerDemo
//
//  Created by Grigory Lutkov on 26.04.15.
//  Copyright © 2015 Grigory Lutkov <Friend.LGA@gmail.com>. All rights reserved.
//

import UIKit

class LeftViewCell: UITableViewCell {

    @IBOutlet var separatorView: UIView!

    override func awakeFromNib() {
        super.awakeFromNib()
        
        backgroundColor = .clear
        
        textLabel!.font = UIFont.boldSystemFont(ofSize: 16.0)
    }
    
    override func layoutSubviews() {
        super.layoutSubviews()
        
        textLabel!.textColor = tintColor
        separatorView.backgroundColor = tintColor.withAlphaComponent(0.4)
    }

    override func setHighlighted(_ highlighted: Bool, animated: Bool) {
        textLabel!.textColor = highlighted ? UIColor(red: 0.0, green: 0.5, blue: 1.0, alpha: 1.0) : tintColor
    }

}
